File: AnnexNoSQL/src/Redis/Prova2.java
package Redis;

import redis.clients.jedis.Jedis;


public class Prova2 {

    public static void main(String[] args) {
        Jedis con = new Jedis("localhost");
        con.connect();
        
        String valor = "Aquesta clau és una clau creada des de Java";
        con.set("clau_Java", valor);
        
        System.out.println(con.get("clau_Java"));
        con.close();
    }
}

File: Tema8_1/src/exemples/Prova2.kt
package exemples

import redis.clients.jedis.Jedis

fun main() {
    val con = Jedis("localhost")
    con.connect()

    val valor = "Aquesta clau és una clau creada des de Kotlin"
    con.set("clau_Kotlin", valor)

    println(con.get("clau_Kotlin"))
    con.close()
}